ui/apps/dashboard/src/app/api/inngest/route.ts

```diff
@@ -0,0 +1,9 @@
+import { serve } from 'inngest/next';
+
+import { inngest } from './client';
+import { runAgentNetwork } from './functions/run-network';
+
+export const { GET, POST, PUT } = serve({
```

Comment:
It doesn't look like our UI's Vercel project has a signing key, so this won't work.

Also, which env are we running this in? I don't think we should use our prod env since that'd necessitate putting its signing key in our UI code

Reply:
Will this be in a non prod environment? Because if it's in prod then it's a massive signing key leak risk. All of the frontend security issues we've handwaved become critical